gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/ManifestBasedDataset.java

```diff
@@ -146,6 +151,13 @@ public Iterator<FileSet<CopyEntity>> getFileSetIterator(FileSystem targetFs, Cop
         }
       }
 
+      // Only set permission for newly created folders on target
```

Comment:
Let's add command here saying if we want to set permission for existing folder, we need to put them in the manifest. Also can you help verify that we support existing folder permission sync in the current job?

Reply:
Yup, added the comment. We should be handling folders in the manifest with https://github.com/apache/gobblin/blob/99c5a76e2b6193a8c78053b4e618ededfccbd3aa/gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/publisher/CopyDataPublisher.java#L195